js/src/elements/form-behavior.ts

```diff
@@ -5,8 +5,13 @@ export const defaultMessageStrings = {
 	inputMaxLength: '{inputLabel} cannot be longer than {maxLength} characters.',
 	inputMinLength: '{inputLabel} must be at least {minLength} characters.',
 	inputRequired: '{inputLabel} is required.',
+	notAuthenticated:
+		'You are not authenticated, please refresh the page and try again. If this issue persists please log out and log back in.',
```

Comment:
I think the first sentence needs to be split into two, and a comma should be added to the final sentence:
```suggestion
		'You are not authenticated. Please refresh the page and try again. If this issue persists, please log out and log back in.',
```